db.go

```diff
@@ -310,11 +332,11 @@ func (db *DB) rollbackJournal(ctx context.Context) error {
 		return err
 	} else if err := dbFile.Close(); err != nil {
 		return err
-	} else if err := journalFile.Close(); err != nil {
-		return err
 	}
 
-	if err := os.Remove(db.JournalPath()); err != nil {
+	if err := journalFile.Truncate(0); err != nil {
+		return err
+	} else if err := journalFile.Close(); err != nil {
 		return err
 	}
```

Comment:
Changed to a truncation as performing a delete while clients are active can cause issues.